src/SFA.DAS.EAS.Web/Views/EmployerManageApprentices/Details.cshtml

```diff
@@ -64,7 +64,8 @@
 
         <h1 class="heading-xlarge" id="learnerName">@Model.Data.FirstName @Model.Data.LastName</h1>
 
-        @{ if (!Model.Data.Status.Equals("Stopped", StringComparison.InvariantCultureIgnoreCase))
+        @{ if (!Model.Data.Status.Equals("Stopped", StringComparison.InvariantCultureIgnoreCase) &&
```

Comment:
It would be better to check the payment status like so:

`(@Model.Data.PaymentStatus == PaymentStatus.Completed)`